File: OOP/OOPA1.py
class Car:
    top_speed = "370 km/h"
    location = 0

    def printTopSpeed(self):
        print("Top Speed: ", self.top_speed)

    def drive(self):
        self.location += 10

    def stop(self):
        print("Car stopped at location: ", self.location)

my_car = Car() 

my_car.drive()
my_car.drive()
my_car.drive()
my_car.stop()
my_car.printTopSpeed()

File: ES2015+/OOPA1.js
class Car{
    constructor(topSpeed){
        this.topSpeed = topSpeed;
        this.location = 0;

}

printTopSpeed() {
    console.log('Top Speed: ${this.topSpeed} mph');
}

drive() {
    this.location += 10;
    console.log('Current Location: ${this.location} miles');
}

stop() {
    console.log('Car stopped. Final location: ${this.location} miles');
}
}

const myCar = new Car(120);
myCar.printTopSpeed();

mycar.drive();
mycar.drive();
